src/services/navigation/useNavLinks/useNavLinks.js

```diff
@@ -55,8 +55,12 @@ export function useNavLinks() {
       isExternalLink: true,
     },
     owner: {
-      path: ({ provider = p, owner = o } = { provider: p, owner: o }) =>
-        `/${provider}/${owner}`,
+      path: ({ provider = p, owner = o } = { provider: p, owner: o }) => {
+        if (provider && owner) {
+          return `/${provider}/${owner}`
```

Comment:
handles the case where `provider` and `owner` are undefined if the user is not logged in